tools/NuGet/template-artifactory/DynamoVisualProgramming.DynamoSamples.nuspec

```diff
@@ -10,11 +10,10 @@
         <icon>content\images\logo.png</icon>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <description>This package bundles all localized Sample files from the Dynamo bin Release folder.</description>
-        <copyright>Copyright Autodesk 2020</copyright>
+        <copyright>Copyright Autodesk 2023</copyright>
     </metadata>
     <files>
-        <!--These files are referenced from the bin directory since they are not harvested for the installer-->
```

Comment:
I deliberately added this comment as the other nuspecs reference files from the current directory without having to reference them from this long relative path. I'm still not clear why there's a difference. See [here](https://github.com/DynamoDS/Dynamo/blob/5634759b89a76b668c1e815f9d3be0de433d2f5b/tools/NuGet/template-nuget/DynamoVisualProgramming.Core.nuspec#L35).